install/jms/bin/ts.jte

```diff
@@ -210,39 +210,39 @@ tslib.name=jmstck
 
 #########################################################################
 # platform.mode is used by to specify runtime environment for testing
-# which can be either javaEE or standalone. For JMS TCK this setting
+# which can be either javaEE or standalone. For Jakarta Messaging TCK this setting
 # is standalone.
 #
-# platform.mode=standalone     Testing JMS TCK as Standalone environment
+# platform.mode=standalone     Testing Jakarta Messaging TCK as Standalone environment
 #########################################################################
 platform.mode=standalone
 
 #######################################################################################
-# JMS RI Implementation Property Settings for Vendor Implementation under test
+# Jakarta Messaging CI Implementation Property Settings for Vendor Implementation under test
 #
-# The TCK uses the following JMS RI Implementations:
-#   - ri	  (For the JMS Reference Implementation or Open Message Queue Product)
+# The TCK uses the following Jakarta Messaging CI Implementations:
+#   - ri	  (for Eclipse Open Message Queue)
 #
 # Here are the common properties that need to be defined for the common implementation
 # functionality:
 #
 # @impl.vi    	This property must be set to the Vendor implementation under test.
 #
-#		Default value is (ri) for the Standalone JMS Reference Implementation 
+#		Default value is (ri) for the Standalone Jakarta Messaging Compatible Implementation 
 #		or the Open Message Queue Product (impl.vi=ri).
 #######################################################################################
 
 impl.vi=ri
 
 # ---------------------------------------------------------------------------------
 # ================================================
-# Standalone JMS Reference Implementation and Open
+# Standalone Jakarta Messaging Compatible Implementation and Open
 # Message Queue Product Configuration Properties
 # ================================================
 #
 # Property settings for ant config.vi/clean.vi tasks to configure, unconfigure, 
-# setup, cleanup, start, and stop the Standalone JMS Reference Implementation 
-# and the Open Message Queue Product for a JMS TCK run.
+# setup, cleanup, start, and stop the Standalone Jakarta Messaging Compatible Implementation 
+# and the Open Message Queue Product for a Jakarta Messaging TCK run.
```

Comment:
In some points, we complement with "Eclipse" the Open Message Queue, like the [line 45](https://github.com/eclipse-ee4j/jakartaee-tck/pull/415/files#diff-34b05b6c2845c35e2d7a9f4be79b0216R45)